modules/apps/depot/depot-service/src/main/java/com/liferay/depot/internal/security/permission/contributor/DepotRoleContributor.java

```diff
@@ -46,37 +46,46 @@ public void contribute(RoleCollection roleCollection) {
 			Group group = _groupLocalService.getGroup(
 				roleCollection.getGroupId());
 
-			if (!group.isDepot()) {
+			if (!group.isCMS() && !group.isDepot()) {
 				return;
 			}
```

Comment:
Hi @AliciaGarciaGarcia ,
I think this statement is no longer needed as both its requirements are checked later in line#55 and line#86. If I'm correct, this was only added to reduce indentation, but we can no longer do that.

Reply:
Can you explain me about the indentation part?
Why can we no longer do that?

Reply:
talked on Slack, indentation reduction is still okay, but we can't use it in our case because of the 2 following if statements